packages/kbn-config/src/deprecation/apply_deprecations.test.ts

```diff
@@ -36,19 +36,16 @@ describe('applyDeprecations', () => {
     const addDeprecation = jest.fn();
     const createAddDeprecation = jest.fn().mockReturnValue(addDeprecation);
     const initialConfig = { foo: 'bar', deprecated: 'deprecated' };
-    const alteredConfig = { foo: 'bar' };
 
-    const handlerA = jest.fn().mockReturnValue(alteredConfig);
-    const handlerB = jest.fn().mockImplementation((conf) => conf);
+    const handlerA = jest.fn().mockReturnValue({ unset: [{ key: 'deprecated' }] });
+    const handlerB = jest.fn().mockReturnValue(undefined);
```

Comment:
Can we add a test (and what should be the behavior) when an invalid command is returned

e.g `unset: [{ key: 'i_do_not_exist'}]`